cdc/generation.cc

```diff
@@ -1157,4 +1157,22 @@ db_clock::time_point get_ts(const generation_id& gen_id) {
     return std::visit([] (auto& id) { return id.ts; }, gen_id);
 }
 
```

Comment:
> When allocating tablets for a CDC table, create the initial CDC stream
> set. We create one stream per each tablet, each stream covering the
> corresponding token range.

I think we should create one stream per shard that has a base table tablet. It means stream updates every time a tablet is migrated (so quite often), but fewer streams overall (100x fewer in the extreme case of a single table).

We can update it post-merge if we decide it's better.